### PR TITLE
Update recommendations and notes regarding microcode and AMD

### PR DESCRIPTION
**Pull request by andrewdavidwong**
Based on discussion with @DemiMarie

CC: @marmarek

### CODE REVIEW

File: user/hardware/system-requirements.md [hunk 1]
```diff
@@ -33,9 +33,13 @@ title: System requirements
...
-  - [Intel VT-x](https://en.wikipedia.org/wiki/X86_virtualization#Intel_virtualization_.28VT-x.29) with [EPT](https://en.wikipedia.org/wiki/Second_Level_Address_Translation#Extended_Page_Tables) or [AMD-V](https://en.wikipedia.org/wiki/X86_virtualization#AMD_virtualization_.28AMD-V.29) with [RVI](https://en.wikipedia.org/wiki/Second_Level_Address_Translation#Rapid_Virtualization_Indexing)
-  - [Intel VT-d](https://en.wikipedia.org/wiki/X86_virtualization#Intel-VT-d) or [AMD-Vi (also known as AMD IOMMU)](https://en.wikipedia.org/wiki/X86_virtualization#I.2FO_MMU_virtualization_.28AMD-Vi_and_Intel_VT-d.29)
+- **CPU:** 64-bit Intel processor (also known as `x86_64`, `x64`, and `Intel 64`)
+  - [Intel VT-x](https://en.wikipedia.org/wiki/X86_virtualization#Intel_virtualization_.28VT-x.29) with [EPT](https://en.wikipedia.org/wiki/Second_Level_Address_Translation#Extended_Page_Tables)
+  - [Intel VT-d](https://en.wikipedia.org/wiki/X86_virtualization#Intel-VT-d)
+  - For security, we recommend processors that are recent enough to still be
+    receiving microcode updates (see [below](#important-updates) for details).
```
Comment (marmarek) [inline user/hardware/system-requirements.md:40]: Maybe worth linking somewhere to the list of support end dates for Intel? https://www.intel.com/content/www/us/en/support/articles/000022396/processors.html (those missing don't have announced date yet, and there is no clear policy how long each CPU family is supported...)

File: user/hardware/system-requirements.md [hunk 2]
```diff
@@ -84,6 +88,49 @@ We recommend consulting these resources when selecting hardware for Qubes OS:
...
+  1. AMD to distribute microcode updates to original equipment manufacturers
+  (OEMs), original design manufacturers (ODMs), and motherboard manufacturers
+  (MB); and
+  2. The user's OEM, ODM, or MB to provide a suitable BIOS or (U)EFI update for
+  the user's system.
+  
+  Historically, AMD has often been slow to complete step (1), at least for its
+  client (as opposed to server) platforms. In some cases, AMD has made fixes
+  available for its server platforms very shortly after a security embargo was
+  lifted, but it did not make fixes  available for client platforms facing the
```
Comment (fepitre) [inline user/hardware/system-requirements.md:119]: double spaces after "fixes"

**Comment by DemiMarie** [comment]
Pinging @andyhhp

File: user/hardware/system-requirements.md [hunk 3]
```diff
@@ -84,6 +88,58 @@ We recommend consulting these resources when selecting hardware for Qubes OS:
...
+  available for its server platforms very shortly after a security embargo was
+  lifted, but it did not make fixes available for client platforms facing the
+  same vulnerability until weeks or months later. (A "security embargo" is the
+  practice of avoiding public disclosure of a security vulnerability prior to a
+  designated date.) By contrast, Intel has consistently made fixes available for
+  new CPU vulnerabilities across its supported platforms very shortly after
+  security embargoes have been lifted.
+
+  Step (2) varies by vendor. Many vendors fail to complete step (2) at all,
+  while some others take a very long time to complete it.
```
Comment (DemiMarie) [inline user/hardware/system-requirements.md:136]:
```suggestion
  Historically, AMD has sometimes been slow to complete step (1), at least for its
  client (as opposed to server) platforms. In some cases
  (notably Speculative Return Stack Overflow, XSA-434/QSB-093),
  AMD has made fixes available for its server platforms very shortly
  after a security embargo was lifted, but it did not make fixes available
  for client platforms facing the same vulnerability until weeks or
  months later. (A "security embargo" is the practice of avoiding public
  disclosure of a security vulnerability prior to a
  designated date.) By contrast, Intel has consistently made fixes available for
  new CPU vulnerabilities across its supported platforms very shortly after
  security embargoes have been lifted.

  Step (2) varies by vendor. Many vendors fail to complete step (2) at all,
  while some others take a very long time to complete it.
```

**Review by DemiMarie — CHANGES_REQUESTED** [review]
This should not be merged until some fact checks have completed.

**Review by DemiMarie — APPROVED** [review]

**Pull request merged**